Connectivity/Classes/Connectivity.swift

```diff
@@ -33,6 +33,10 @@ public class Connectivity: NSObject {
 
     /// Optionally configure a bearer token to be sent as part of an Authorization header.
     public var bearerToken: String?
+    
+    /// Optionally configure the Authorization header manually.
+    /// Note: `bearerToken` has a higher priority
```

Comment:
I've actually changed this around so that the authheader has higher priority - since the bearer token header is just a more specific instance of this I figured this probably has higher utility (see comment below).